javascript/javascript3/week3/shoppingCart.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Users repositories</title>
+
+</head>
+
+<body>
+
+
+
+    <script>
+        class Product {
+            constructor(name, price) {
+                this.name = name;
+                this.price = price;
+            }
+
+            convertToCurrency(currency) {
+                const exchangeCurrency = currency.toUpperCase();
+                fetch(
+                        `https://free.currconv.com/api/v7/convert?q=DKK_${exchangeCurrency}&compact=ultra&apiKey=887a84c9cd061cfbd5fa`
+                    )
+                    .then((response) => response.json())
+                    .then((data) => {
+                        const exchangeRate = Object.values(data)[0];
+                        const exchangeValue = (this.price * exchangeRate).toFixed(2) + currency;
+                        console.log(exchangeValue);
+                    })
+                    .catch((error) => {
+                        throw error;
+                    });
+            }
+        }
+
+        class ShoppingCart {
+            constructor() {
+                this.products = [];
+            }
+
+            addProduct(product) {
+                this.products.push(product);
+            }
+
+            removeProduct(product) {
+
+                const index = this.products.indexOf(product);
+                if (index > -1) {
+                    this.products.splice(index, 1);
+                }
+            }
+
+            searchProduct(productName) {
+                const index = this.products.indexOf(productName);
+                let searchedProducts = [];
+                if (index > -1) {
+                    searchedProducts.push(this.products[index]);
+                }
+                return (this.searchedProducts = searchedProducts);
+            }
+
+            getTotal() {
+                let totalAmount = 0;
+                this.products.forEach((product) => (totalAmount += product.price));
+                this.totalAmount = totalAmount;
+            }
+
+            renderProducts() {
+                this.products.map((product) => {
+
+                    console.log(product.name);
+                    console.log(product.price);
+
+
+
+
+                });
+            }
+
+
+
+            async getUser() {
+                try {
+                    const use = await fetch("https://jsonplaceholder.typicode.com/users/1");
```

Comment:
user `1` is hard-coded. It should be a parameter to the function. `getUser(user)`